piker/clearing/_paper_engine.py

```diff
@@ -463,7 +463,7 @@ async def trades_dialogue(
         # TODO: load paper positions per broker from .toml config file
         # and pass as symbol to position data mapping: ``dict[str, dict]``
         # await ctx.started(all_positions)
-        await ctx.started({})
+        await ctx.started(({}, {'paper',}))
```

Comment:
My last question is maybe this should just be an explicit map?

```python
{'positions': {}, 'accounts': {'paper',}}
```

kinda thing?